test/sanity/ignore.txt

```diff
@@ -3366,12 +3366,12 @@ lib/ansible/modules/network/eos/eos_logging.py validate-modules:doc-required-mis
 lib/ansible/modules/network/eos/eos_logging.py validate-modules:missing-suboption-docs
 lib/ansible/modules/network/eos/eos_logging.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/network/eos/eos_logging.py validate-modules:undocumented-parameter
-lib/ansible/modules/network/eos/eos_static_route.py validate-modules:doc-choices-do-not-match-spec
```

Comment:
Hey Gomathi, here we shouldn't just move the old module ignore to the new ones. I guess you'll have to do some changes over the PR but the old module should just be marked as deprecated *and* moved to _ (i.e eos_static_route.py to _eos_static_route.py). The the ignores should be moved accordingly.

Reply:
Corrected!